hypertrace-view-creator/src/main/resources/configs/service-call-view/application.conf

```diff
@@ -39,9 +41,13 @@ pinot.streamConfigs =
     stream.kafka.consumer.factory.class.name: "org.apache.pinot.plugin.stream.kafka20.KafkaConsumerFactory",
     stream.kafka.decoder.class.name: "org.apache.pinot.plugin.inputformat.avro.confluent.KafkaConfluentSchemaRegistryAvroMessageDecoder",
     stream.kafka.decoder.prop.schema.registry.rest.url: "http://schema-registry-service:8081",
+    stream.kafka.decoder.prop.schema.registry.rest.url: ${?SCHEMA_REGISTRY_URL},
     stream.kafka.hlc.zk.connect.string: "zookeeper:2181",
+    stream.kafka.hlc.zk.connect.string: ${?ZK_CONNECT_STR},
     stream.kafka.zk.broker.url: "zookeeper:2181",
+    stream.kafka.zk.broker.url: ${?ZK_CONNECT_STR},
```

Comment:
@kotharironak isn't there a way to read from ENV and if not exists, default to the given value? Duplicating so many properties doesn't look clean and is error prone.

Reply:
This is what I've been preaching. You could have it in the code but that's also messy. If you 100 ways to configure something, the code will be 100 times messier :)